File: src/main/java/Guest.java
public class Guest {
    private String name;

    public Guest(String name) {
        this.name = name;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;

        Guest guest = (Guest) o;

        return name != null ? name.equals(guest.name) : guest.name == null;

    }

    @Override
    public int hashCode() {
        return name != null ? name.hashCode() : 0;
    }
}

File: webui/src/main/javascript/Guest.js
import React from 'react';

 class Guest extends React.Component {

    constructor(props) {
        super(props);
        this.state = {name: props.name};
    }

    render() {
        return <div className={"guest" + this.state.name} style = {{background: "grey", margin: "5px", padding: "5px"}}>{this.state.name}</div>
    }

 }

 export default Guest;